dashboard/src/shared/types.ts

```diff
@@ -269,7 +273,7 @@ export interface IPackageManifest extends IResource {
 export interface IClusterServiceVersionCRDResource {
   kind: string;
   name: string;
-  version: string;
+  version?: string;
```

Comment:
Why isn't this required for the resource?

Reply:
actually it's required but it was not being used, re-added